pumpkin/Cargo.toml

```diff
@@ -4,6 +4,11 @@ version = "0.1.0-dev"
 description = "Empowering everyone to host fast and efficient Minecraft servers."
 edition = "2021"
 
+[package.metadata.winresource]
+FileDescription = "Pumpkin"
+OriginalFilename = "pumpkin.exe"
+LegalCopyright = "Copyright © 2024 Aleksander Medvedev"
+
```

Comment:
Consider adding the FileVersion, I would make it the same as `package.version`

Reply:
winresource automatically sets FileVersion.